src/pyproject.toml

```diff
@@ -8,11 +8,12 @@ requires = [
     'wheel >=0.36.2',
     'cypari2 >=2.1.1',
     'cysignals >=1.10.2',
-    # Exclude 3.0.3 because of https://github.com/cython/cython/issues/5748
-    'cython >=3.0, != 3.0.3, <4.0',
+    # per https://github.com/scipy/scipy/blob/maintenance/1.13.x/pyproject.toml
```

Comment:
Could you please revert these changes to `pyproject.toml`. This file defines the version constraints of building the sagelib project, and for this it doesn't matter what constraints scipy has for building itself.

Reply:
I've given you the rights to push directly to my fork. Please accept the invite and fix what's needed - I'm not 100% sure I get what you'd exactly want to change.